### PR TITLE
Feature/printing

### PR DESCRIPTION
**Pull request by Dremars**
New printing tool for ToolMenu. Printing tool opens separate view on top of the current one with correct map state for printing purposes.

Updated Leaflet library

### CODE REVIEW

**Comment by Dremars** [comment]
[(3) Tulostaminen](https://trello.com/c/YUTFfS0v/68-3-tulostaminen)

**Comment by Dremars** [comment]
Need to add translations before merging

File: src/views/PrintView/PrintView.js [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+                    const { name, street_address: address } = description.units[0];
+                    return (
+                      <React.Fragment key={description.number}>
+                        <StyledTableRow key={description.number}>
+                          <StyledTableCell>{description.number}</StyledTableCell>
+                          <StyledTableCell>
+                            {
+                              name
+                              && (
+                                <Typography variant="subtitle1">
```
Comment (mikkoisa) [inline src/views/PrintView/PrintView.js:295]: The unit name seems to be h6, which is confusing for screen readers. These could be all h1 or we could add a single main title h1 to the top of the page ("Printing view" for example) and these would be h2.

**Review by mikkoisa — CHANGES_REQUESTED** [review]

File: src/views/PrintView/PrintView.js (hunk 1 above)
Comment (Dremars) [reply src/views/PrintView/PrintView.js:296]: Those were meant to be paragraphs. It seems variant="subtitle1" change component to `h6`. Changing component to `p`

**Review by mikkoisa — APPROVED** [review]

**Pull request merged**